### ISSUE TITLE
kent logs all http requests

### ISSUE THREAD

**Issue by willkg** [willkg/kent#69]
Currently, kent logs all http requests. That's fine, except if you're using docker compose health check at 1s and then kent spits out something like this every second:

```
tecken-fakesentry-1  | [2024-01-31 00:17:27,798] INFO: _internal: 127.0.0.1 - - [31/Jan/2024 00:17:27] "GET / HTTP/1.1" 200 -
```

That's unhelpful.

We should rework logging so it's only logging the things that are useful. Namely sentry event payload handling.

**Comment by willkg** [comment]
relud suggests adding a logging filter:

https://stackoverflow.com/questions/62000942/flask-block-specific-endpoints-from-logging/62001693#62001693

That'd also work, but I think reworking logging is better in the long run.

**Comment by willkg** [comment]
Fixed in #70.
